pages/1.10/security/ent/tls-ssl/ca-custom/index.md

```diff
@@ -8,26 +8,310 @@ excerpt:
+
+
+# Requirements
+
+In order to install DC/OS Enterprise with a custom CA certificate you will need:
+
+- to use the [advanced DC/OS installation method](/1.10/installing/custom/advanced/). Other installation methods are not supported.
+- A file containing the custom CA certificate.
+- A file containing the private key associated with the custom CA certificate.
+- If the CA is **not** a self-signed root CA, a file containing the certificate chain associated with the custom CA certificate. 
+
+## Manually placing the custom CA certificate, the associated private key and the certificate chain onto the bootstrap node
+The custom CA certificate, the associated private key and the certificate chain files must be put in the `$DCOS_INSTALL_DIR/genconf/` directory on the bootstrap node:
+
+```bash
+cd $DCOS_INSTALL_DIR
+ls genconf/
+```
+```bash 
+dcos-ca-certificate.crt
+dcos-ca-certificate-key.key
+dcos-ca-certificate-chain.crt
+```
+
+## Manually placing the private key associated with the custom CA certificate onto the master nodes
+
+For security reasons, the installer will not copy the private key from the bootstrap node to the master nodes. 
+The private key associated with the custom CA certificate must be distributed manually to every DC/OS master node **before starting the installation**. 
+
+The filesystem path for the private key file must be `/var/lib/dcos/pki/tls/CA/private/custom_ca.key`. 
+The directory `/var/lib/dcos/pki/tls/CA/private/` can be created manually with the following  command before putting the file `custom_ca.key` in the directory on every DC/OS master node:
+
+```bash
+ mkdir -p /var/lib/dcos/pki/tls/CA/private
+```
+
+Furthermore, the file containing the private key `custom_ca.key` corresponding to the custom CA certificate must be owned by the root Unix user and have 0600 permissions set.
+
+If you copy private key file over the network onto the master nodes, the network channel must be adequately protected.
+An example of copying the CA private key is given below. The commands are executed on the bootstrap node. The `W.X.Y.Z` denotes the IP address of a master node below.
+
+```bash
+cd $DCOS_INSTALL_DIR/genconf
+scp dcos-ca-certificate-key.key centos@W.X.Y.Z:/var/lib/dcos/pki/tls/CA/private/custom_ca.key
+```
+
+## Specifying the locations of the custom CA certificate, the associated private key and the certificate chain files in the DC/OS configuration file
+
+The filesystem paths to the custom CA certificate, associated private key and certificate chain files in the `$DCOS_INSTALL_DIR/genconf/` directory on the bootstrap node must be specified in the DC/OS configuration file using, respectively, the `ca_certificate_path`, `ca_certificate_key_path` and  `ca_certificate_chain_path` parameters.
+The paths to the custom certificate, the associated private key and the certificate chain must be relative paths. (without the `$DCOS_INSTALL_DIR` prefix)
```

Comment:
Let us please write "custom CA certificate" instead of "custom certificate", for consistency.

Reply:
Instead of correcting that, let us simplify.

Instead of 

> The paths to the custom certificate, the associated private key and the certificate chain must be relative paths. (without the `$DCOS_INSTALL_DIR` prefix)

let us write

> The file paths must be relative to `$DCOS_INSTALL_DIR`

In the Google Doc in the Glossary we added `This is the current working directory when executing dcos_generate_config.ee.sh.` to the explanation of "Installation directory". Where and why did this get lost? Was this intentional?

Reply:
@jgehrcke I rephrased the explanation `This is the current working directory when executing dcos_generate_config.ee.sh.` in the **Installation** section. I moved it into that section in response to Jon's remark that the glossary terms normally define concepts and do not specify technical details. Do you want me to move it back to the glossary in its original form?
To me this important information looks equally ok in both sections as long as it is not removed from the document.